### DISCUSSION TITLE
Community soft fork

### DISCUSSION THREAD

**Discussion by creatorrr** [cozodb/cozo#278] [Ideas]
Starting a soft fork of cozo here:

https://github.com/cozo-community/cozo

This is not intended to be a hard fork coz Ziyang (@zh217) has indicated that he continues to support it albeit with limited time.

Discord: https://discord.gg/DMHUyG2RFA

**Comment by creatorrr** [comment]
Would appreciate volunteers and help for maintaining

**Reply by boticello** [reply]
Great to hear that development on Cozo will continue.

**Comment by andrewbaxter** [comment]
I'd be really happy to see this maintained too, although I lack the expertise to do it myself.  What does this mean for opening PRs? I just made #282 for example

**Reply by creatorrr** [reply]
I think @zh217 does frequently review and merge PRs and the fork will keep in sync with upstream so it should be fine to open PRs here. That said- we are setting up a CI/CD pipeline for building and publishing packages automatically. And since there's more than one maintainer on the community fork, it might get merged faster. :)
